src/Template/Element/sql_log_panel.ctp

```diff
@@ -63,7 +63,7 @@ $noOutput = true;
             <tbody>
                 <?php foreach ($queries as $query): ?>
                 <tr>
-                    <td><?= h($query['query']) ?></td>
+                    <td><pre><?= SqlFormatter::format($query['query'], false) ?></pre></td>
```

Comment:
OK. Given that @lorenzo thinks adding syntax highlighting is important, I'll be adding that tonight.